bitnami/redis/templates/configmap-scripts.yaml

```diff
@@ -23,14 +23,16 @@ data:
         fi
     }
 
+    NODES_ALIVE="$(getent hosts "{{ template "redis.fullname" . }}-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}")"
+
     export REDIS_REPLICATION_MODE="slave"
     if [[ $HOSTNAME =~ (.*)-([0-9]+)$ ]]; then
-      if [[ ${BASH_REMATCH[2]} == "0" ]]; then
+      if [[ -z "$NODES_ALIVE" ]]; then
```

Comment:
You can also use the exit code of the `getent hosts` command to decide whether there are alive nodes or not.

Reply:
Hm. Good idea. Will apply.

Reply:
It appears that this did not work out. It appears that the query will succeed when starting the sentinel because the first pod itself is already registered in DNS. Replaced with 
[[ -z "$(getent ahosts "$HEADLESS_SERVICE" | grep -v "^$(hostname -i) ")" ]]
(list all matching records, remove those that have my own IP).